checkov/sca_package_2/output.py

```diff
@@ -141,10 +145,20 @@ def create_cli_output(fixable: bool = True, *cve_records: list[Record]) -> str:
                             parsed_version = packaging_version.parse(root_package_fix_version.strip())
                             fix_versions_lists.append([parsed_version])
 
-                    root_package_lines = validate_lines(record.vulnerability_details.get("root_package_file_line_range"))
+                    root_package_lines = validate_lines(
+                        record.vulnerability_details.get("root_package_file_line_range"))
                     if lines or root_package_lines:
                         lines_details_found_cves = True
 
+                    if record.vulnerability_details and record.vulnerability_details.get("risk_factors", {}).get(
+                            "IsUsed"):
+                        cve_count.used += 1
```

Comment:
according to this logic, you are counting by cve, and not by package.
make sure it is the expected behavior